app/serializers/application.js

```diff
@@ -1,3 +1,29 @@
-import { ActiveModelSerializer } from 'active-model-adapter';
+import { ActiveModelSerializer } from "active-model-adapter";
+import _ from "lodash";
 
-export default ActiveModelSerializer.extend();
+// @TODO: rename the item model into package and remove those aliases
+const ALIASES = {
+  package: "item",
+  packages: "items"
+};
+
+function normalize(payload) {
```

Comment:
👍

Reply:
@abulsayyad123 I think we can remove the normalization in the `orders_package` serializer, since it inherits from here. This applies to everything. Haven't done it for now but thinking about it

Reply:
i had the same question regarding `orders_packages` serializer. Can we remove it ? I thought it was part of separate entity, so application serializer wouldn't change it.

Reply:
I believe we can : )